oomox_gui/theme_model.py

```diff
@@ -358,7 +358,7 @@ def merge_theme_model_with_plugins(theme_model_name, base_theme_model=None):
         'fallback_key': 'SEL_BG',
         'display_name': _('Accent Color'),
         'value_filter': {
-            'TERMINAL_THEME_MODE': ['basic', ],
+            'TERMINAL_THEME_MODE': ['basic', 'manual', ],
```

Comment:
instead of using `TERMINAL_ACCENT_COLOR` please just use `SEL_BG` directly in the plugin

if that doesn't work for you - please let me know - i'll help or fix that

Reply:
SEL_BG isnt part of the terminal model and this way i think works better as thats what it is also SEL_BG can then be a different col,or then accent color

Reply:
treat theme model as one common - its separation was done mainly for the gui and generally to make it easier looking some values

but the meaning of Terminal Accent Color - is main terminal color for generating terminal colorscheme, when other colors are not provided,  using basic generation algo (that's why it's not displayed for other terminal theme modes)

Reply:
ah, btw, just one side-note - dont forget what in each plugin you also free to define addtional, plugin-specific colors in the theme model

Reply:
can i have an example of this please?

Reply:
of extra colors?

Reply:
ya :)

Reply:
https://github.com/themix-project/oomox/blob/pre-1.14/oomox_gui/plugin_api.py#L127

https://github.com/themix-project/themix-plugin-base16/blob/453cccbd2e22f69078c6ded894421c4c0f943bc0/oomox_plugin.py#L355-L397

Reply:
cheers mate

Reply:
can this PR be closed then or you still have some unresolved problem with your new plugin?